school_lunch/models/models.py

```diff
@@ -19,7 +19,7 @@ class Menu(models.Model):
     _description = "Daily Menu"
     _order = "date desc, meal_type"
 
-    name = fields.Char("Meal Title", required=True)
+    name = fields.Char("Meal Title", required=True, default="")
```

Comment:
Or you can use `_(New Menu)`  instead of an empty string when it is not set yet like I've seen some examples in standard... but don't write it to menu.name, e.g.:
`
@api.depends("name", "date")
    def _compute_display_name(self):
        for menu in self:
            if not menu.name:
                name = _(New Menu)
            if self.env.context.get("display") == "simple":
                menu.display_name = name
```
etc

Btw I don't think `self.env.context.get("display", "date") == "simple"` does anything else than using `self.env.context.get("display") == "simple"`. And the string concatenation could be improved using f-strings.

Reply:
Indeed, the `self.env.context.get("display", "date") == "simple"` could spare the `date` element.